_maps/map_files/stations/cerestation.dmm

```diff
@@ -91270,6 +91343,11 @@
 	amount = 5
 	},
 /obj/structure/table/glass,
+/obj/machinery/camera{
+	c_tag = "Virology South";
+	dir = 8;
+	network = list("SS13","QM")
```

Comment:
This should be "CMO", not "QM"

Reply:
Another PR remapped medbay, these cameras are no longer required